tests/integration/telemetry/stats/prometheus/nullvm/dashboard_test.go

```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package telemetry
+package nullvm
```

Comment:
Is this the first part of a larger refactor ?  This test shouldn't really care about `nullvm` (and we want it to work across vms) right?

Reply:
Yes, want to see if this reduces runtime first and then lets discuss the exact directory structure we want to keep!

Reply:
Doug, for a much larger refactor we need a better way of applying our configs..
For now I have just moved dashboard test. 

The nullvm in directory structure here refers to the fact that prometheus is not deployed using wasm vm. Should we rather call it default?

Reply:
i think nullvm is fine for now.